### DISCUSSION TITLE
Suggestions for address input

### DISCUSSION THREAD

**Discussion by FilipHarald** [scaffold-eth/scaffold-eth-2#506] [Ideas]
Instead of having to look around the UI for the address, you could instead get a drop-down for addresses that's relevant. I've [implemented a PoC here](https://github.com/FilipHarald/scaffold-eth-2-sandbox/tree/feat/address_input_suggestions) and could clean it up a little and make a PR.

https://github.com/scaffold-eth/scaffold-eth-2/assets/6756284/090f9565-6a0b-4665-b468-fe61de1da7d6

**Comment by FilipHarald** [comment]
Probably no one will ever read this. But I made an extension for it now :)
https://github.com/FilipHarald/MagicAddressInput

**Reply by damianmarti** [reply]
Nice!! There will be a SE-2 Extensions hackathon soon! Stay tuned!
